tutorials/pub-sub/README.md

```diff
@@ -331,7 +331,55 @@ dapr stop --app-id react-form
 
 <!-- END_STEP -->
 
-4. If you want to deploy this same application to Kubernetes, move onto the next step. Otherwise, skip ahead to the [How it Works](#How-it-Works) section to understand the code!
+4. If you want more insights into the cloud events being published on Redis streams, move onto the next section. If you want to deploy this same application to Kubernetes, move onto the section after that. Otherwise, skip ahead to the [How it Works](#How-it-Works) section to understand the code!
```

Comment:
```suggestion
4. If you want more insights into the Cloud Events being published on Redis streams, keep going.  If you want to deploy this same application to Kubernetes,  jump to the [Run in Kubernetes](#Run-in-Kubernetes). Otherwise, skip ahead to the [How it Works](#How-it-Works) section to understand the code!
```